english/experience/black-point/black-point.md

```diff
@@ -4,4 +4,6 @@ However, nothing happen, it's just a little black point.
 
 What do you do now?
 
-[Open your eyes](../../marshmallow.md)
\ No newline at end of file
+[Open your eyes](../../marshmallow.md)
+
+[Concentrate harder on the black point] (/stargaze.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.